migrations/migrations/20200117201132-user-model.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('user', {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      createdAt: {
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        type: Sequelize.DATE
+      },
+      firstName: {
```

Comment:
We use camelCase in code and columns in snake_case for database, check option above for more detail